specs/routes.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require('supertest');
+const express = require('express');
+const port = 3000;
+
+
+describe('/api/img_carousel/:id', function() {
+  const id = Math.floor(Math.random() * (100 - 1) + 1);
```

Comment:
don't think you need to randomize. feel like the art of testing is having a nose for what might go wrong and to me at least this seems like an unlikely source of failure. don't need to change either though and it does make the test more robust, just my thought process.